src/confcom/azext_confcom/tests/latest/test_confcom_fragment.py

```diff
@@ -939,35 +933,1235 @@ def test_fragment_vn2_workload_identity_mounts(self):
+                import_statement.get(config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_ISSUER,""),iss
+            )
+            self.assertEqual(
+                import_statement.get(config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_FEED,""),feed
+            )
+            self.assertEqual(
+                import_statement.get(config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_MINIMUM_SVN,""), "1"
+            )
+            self.assertEqual(
+                import_statement.get(config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS_INCLUDES,[]),[config.POLICY_FIELD_CONTAINERS, config.POLICY_FIELD_CONTAINERS_ELEMENTS_REGO_FRAGMENTS]
+            )
+
+        except Exception as e:
+            raise e
+        finally:
+            delete_silently(filename)
+            delete_silently(out_path)
+            delete_silently(temp_filename)
+
+    def test_remote_fragment_references(self):
+        filename = "payload6.rego"
+        filename2 = "payload7.rego"
+        first_fragment = "first_fragment.json"
+        fragment_json = "fragment_remote.json"
+        feed = f"{self.registry}/test_feed:v1"
+        feed2 = f"{self.registry}/test_feed2:v2"
+        out_path = filename + ".cose"
+        out_path2 = filename2 + ".cose"
+
+        # fragment_text = self.aci_policy.generate_fragment("payload6", 1, OutputType.RAW)
+
+        try:
+            write_str_to_file(first_fragment, self.custom_json)
+            write_str_to_file(fragment_json, self.custom_json2)
+            acifragmentgen_confcom(
+                None, first_fragment, None, "payload7", "1", feed, self.key, self.chain, None, output_filename=filename
+            )
+
+            # this will insert the import statement from the first fragment into the second one
+            acifragmentgen_confcom(
+                None, None, None, None, None, None, None, None, generate_import=True, minimum_svn="1", fragments_json=fragment_json, fragment_path=out_path
+            )
+
+            push_fragment_to_registry(feed, out_path)
+
+            acifragmentgen_confcom(
+                None, fragment_json, None, "payload7", "1", feed2, self.key, self.chain, None, output_filename=filename2
+            )
+
+            # make sure all of our output files exist
+            self.assertTrue(os.path.exists(filename2))
+            self.assertTrue(os.path.exists(out_path2))
+            self.assertTrue(os.path.exists(fragment_json))
+            # check the contents of the unsigned rego file
+            rego_str = load_str_from_file(filename2)
+            # see if the import statement is in the rego file
+            self.assertTrue(feed in rego_str)
+            # make sure the image covered by the first fragment isn't in the second fragment
+            self.assertFalse("mcr.microsoft.com/acc/samples/aci/helloworld:2.9" in rego_str)
+        except Exception as e:
+            raise e
+        finally:
+            delete_silently(filename)
```

Comment:
Can we make delete silently take a list of paths to delete? It will make this look cleaner.